release-calendar/src/client/components/fakeData.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const FAKEEVENTS = [
```

Comment:
How hard is it to use fake release data instead of these fake events? Can we use `development-data.ts` instead? That might involve running the server bundle every time.

Other comment - this file doesn't belong in `components/` because it's not a component.

Reply:
I planning to create another layer between the components and the database that forms data the way I need it. For example, the json FAKEEVENTS are pre-divided into channels. I can add a TODO about this and for now add a different folder for fakeData to all live in.

Reply:
Let's not call it `fakeData` or `FAKEEVENTS` then. You can set the values to your test data and add a TODO to hook it up with the API layer.